posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "Liberty Tools, Maven plugin, and Gradle plugin updates for Java developers"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yeekangc
+author_github: https://github.com/yeekangc
+seo-title: Updates in Liberty Tools and Liberty build plugins for Java developers - OpenLiberty.io
+seo-description: New releases of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin for Java developers to easily write applications in Eclipse IDE, Visual Studio Code, and IntelliJ IDEA and build them using Maven and Gradle.
+blog_description: "New releases of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin for Java developers to easily write applications in Eclipse IDE, Visual Studio Code, and IntelliJ IDEA and build them using Maven and Gradle."
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-marketplaces.png
+open-graph-image-alt: Liberty Tools for Eclipse IDE, Visual Studio Code, IntelliJ IDEA
+---
+= Liberty Tools, Maven plugin, and Gradle plugin updates for Java developers
+YK Chang <https://github.com/yeekangc>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// // // // // // // //
```

Comment:
I would delete all these commented lines, just for tidiness as they're not needed any more.